pkgs/tools/security/vaultwarden/webvault.nix

```diff
@@ -41,12 +41,20 @@ buildNpmPackage rec {
   '';
 
   nativeBuildInputs = [
-    (python3.withPackages (ps: [ ps.setuptools ]))
+    python3
   ];
 
   makeCacheWritable = true;
 
-  ELECTRON_SKIP_BINARY_DOWNLOAD = "1";
+  env = {
+    ELECTRON_SKIP_BINARY_DOWNLOAD = "1";
+    npm_config_build_from_source = "true";
+  };
+
+  npmRebuildFlags = [
+    # FIXME one of the esbuild versions fails to download @esbuild/linux-x64
+    "--ignore-scripts"
+  ];
```

Comment:
that's because it is not in the lockfile. No idea why....

Reply:
It's an optional dependency

Reply:
If I understand the lock file format correctly, then it should be in there, too but I could also be wrong